src/components/BorrowerProfile/BorrowerName.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<div class="borrower-name-wrapper">
+		<h2>{{ name }}</h2>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		name: {
+			type: String,
+			default: ''
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.borrower-name-wrapper {
+	position: absolute;
+	width: 160px;
+	height: 48px;
+	left: 336px;
+	top: 188px;
+	color: $umbreon;
+
```

Comment:
@mlibazisi for these classes we need to use the [Tailwinds css classes](https://tailwindcss.com/docs). We don't want any scss in this file. 

for example the color css above would be handled with this tailwind class "text-gray-800"

Reply:
The same is true for all of these other css definitions, you need to find tailwind specific classes and delete all css definitions in this file.

Reply:
👍